network/load-balancer/concepts.mdx

```diff
@@ -169,6 +169,16 @@ Read more about HTTP/2 and HTTP/3 and how they can be configured on Scaleway Loa
 
 <Concept>
 
+## Kubernetes Load Balancer
+
+When you create a `LoadBalancer` service within your [Scaleway Kubernetes cluster](/containers/kubernetes/quickstart/), the Scaleway Cloud Controller Manager will spin up an external Scaleway Load Balancer with the correct configuration to expose the specified service in your cluster.
```

Comment:
@RoRoJ this is weird but the staging doc shows an "and" after the first coma ...
![Capture d’écran 2023-11-06 à 18 04 34](https://github.com/scaleway/docs-content/assets/109532009/2ac7e9b1-085b-49eb-bcad-ebb64c5b4961)

Reply:
As above, this has been fixed since we deployed the staging